getting_started.md

```diff
@@ -0,0 +1,104 @@
+
+**Note:** If you've already got a friend on Scuttlebutt, ask them what pub(s) they follow. Otherwise it's possible that you'll be out of range of each other, so to speak. 
+
+The pub's you're currently connected to are listed on the left side if you click on "Public".
+
+## Getting all the Gossip
+
+As soon as you connect to a pub or start following a person Scuttlebutt will start syncing down all the "gossip" they have for you, and pass on any "gossip" you have for them. *The first time you connect to a pub server there will be a lot to download. * So, give it some time.
+
+## Get Social!
+
+You'll see some channels listed in "Active channels" on the left, but the "More" menu up at the top has a "Browse All" option under "Channels".
+
+![browse all button](assets/patchwork-browse-all.png)
+
+Find a topic that looks interesting and start reading. When you're ready to "Publish" remember that it uses Markdown formatting. 
+
+Remember, it doesn't matter if you're online or not. You'll still see all the images, and you'll be able to respond to posts and create new ones. The next time you connect to a local person or a pub server everything will connect up. When you're online this will happen constantly, and responses can appear very quickly. 
+
+### Some places to start
+
+Once onboard patchwork, check out the following channels by typing their name (with #) in the search bar:
+
+* #new-people: introduce yourself
+* #faq: first impressions, what is confusing as a new user?
+* #patchwork: report bugs, suggestions, etc
+* #scuttlebutt: protocol related discussion
+
+
+**NOTE:** There is no edit. There is no Undo. There is no delete. So, use your spellcheck, actually look at the Preview (When 	and publish mad.
+
+## Linking To Things
+Sooner or later you're going to want to link to some web page. With markdown formatting that's just `[display text](destination url)` 
+
+How do you link to other messages _in_ Scuttlebutt though? Well, first you need the message id. Right click on a link to the message you want to link to and choose "Copy Message ID" Then make a link just like a normal url link but use the message id as the destination url. 
+
+Profiles are a little different. A profile is a feed for a single person. So, in that case you want to use "Copy Link Ref" For example, if I wanted to link to André's profile I'd say `[here's André's profile](@QlCTpvY7p9ty2yOFrv1WU1AE88aoQc4Y7wYal7PFc+w=.ed25519)`  
+
+### Sharing Scuttlebutt things with other people
+I wanted to share a really interesting conversation with a friend of mine who wasn't on Scuttlebutt yet, so I right clicked on the first message in the thread, chose "Copy External Link" and then messaged them that link. You could Tweet it or post it on Facebook, or whatever.  The link goes to a public site where you can see, but not interact with, posts on Scuttlebutt.
```

Comment:
maybe mention this is all possible by the good grace of @clehner hosting a central server. eventually hope that more pubs will have external portals by default.

Reply:
dunno, feels a little off topic. I've got no trouble giving someone props for something like that but it seems a little odd to put here (to me). You folks own the repo. You get to make the call.

side note: i looked into setting one up but saw no docs on it. I saw a commit message that looked like that ability had been added to the pub server, but wasn't sure if that was correct and if so how to turn it on, or if it was just automatically on by default if you set up a pub server or....  People don't create docs for the tools they write and then wonder why others don't start using them... :/

Reply:
sorry, i meant less of giving props, more of setting expectations for why the external portal running on a single vps might be down.